### PR TITLE
Hoods for Winter Coats

### PR DESCRIPTION
**Pull request by Gotimanga**
## About the PR
Added hoods for most of the winter coats (Excluding Musician, Chef, and Clown). Hoods are mainly cosmetic additions, however do slow the player cooling in freezing environments slightly.  This is one of the first times I have opened a 2d editor, so feel free to recommend any changes or issues you have with any sprites.

image

## Media

https://github.com/space-wizards/space-station-14/assets/127038462/eeebbbbd-5aa5-4579-a84a-02f77fcf98bf

https://github.com/space-wizards/space-station-14/assets/127038462/6f392c5e-30ed-472f-a461-cf76c279d11c

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- added: Winter coats now have hoods

### CODE REVIEW

**Comment by DogZeroX** [comment]
PR of the year

**Comment by lzk228** [comment]
there's definitely should be sound when you equip it

**Comment by Gotimanga** [comment]
I completely forgot about that, that's a good idea. I'll work on it when my IDE stops throwing errors whenever I run it.

**Comment by potato1234x** [comment]
why not use the sprites from tg

**Comment by potato1234x** [comment]
specifically [these](https://github.com/tgstation/tgstation/blob/d1582bf1b41a0e24fd7a18124c6ff46b86b393f9/icons/mob/clothing/head/winterhood.dmi)

**Comment by VektorZ1** [comment]
Cool

**Comment by Gotimanga** [comment]
@potato1234x I didn't actually know TG already had sprites, and I kind of wanted to try make my own anyway. Is it the sides of the hood that you think are too big?

**Comment by potato1234x** [comment]
The fluff parts are too thick, theyd look better consistently 2 px wide

**Comment by Gotimanga** [comment]
2 px seems a bit too small for me, however I could try to make the hoods all be consistent 3 px like the hood on the right of the picture.
image

**Comment by Partmedia** [comment]
> there's definitely should be sound when you equip it

Funny, when I equip my hood IRL I don't hear a sound?

(This would be fine to add, and you're welcome to add this to this PR, but it's fine if this PR is just the sprite changes.)

**Comment by Gotimanga** [comment]
I assigned some sounds to the action in a recent commit, its just the bag rustling sound, but I think it does the job.

**Comment by potato1234x** [comment]
> 2 px seems a bit too small for me, however I could try to make the hoods all be consistent 3 px like the hood on the right of the picture. image

yeah that looks good

**Comment by mirrorcult** [comment]
it probably would have made more sense to use the tg hood sprites from the beginning but since these are already sprited i think they look good

File: Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml [hunk 1]
```diff
@@ -33,63 +33,88 @@
...
+  id: ClothingOuterWinterCoatToggleable
+  name: winter coat with hood
+  noSpawn: True
+  components:
+  - type: ToggleableClothing
+    clothingPrototype: ClothingHeadHatHoodWinterDefault
+    slot: head
+  - type: ContainerContainer
+    containers:
+      toggleable-clothing: !type:ContainerSlot {}
```
Comment (LankLTE) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml:45]:
```suggestion
  - type: ContainerContainer
    containers:
      toggleable-clothing: !type:ContainerSlot {}
      storagebase: !type:Container
        ents: []
```

You're overwriting the containercontainer component from the ClothingOuterStorageBase. This fixes test fails.

Comment (Gotimanga) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml:45]: Thank you so much

**Comment by Gotimanga** [comment]
Ok so maybe I wrote "added" instead of "add" for the CL and that's why its not showing up in the changelog when I open it on my local repo. Is there any way to fix this?

**Comment by deltanedas** [comment]
make a pr fixing the cl

**Comment by Gotimanga** [comment]
just a pr with nothing but a CL?

**Pull request merged**